### PR TITLE
Enchilada manual display card

### PR DESCRIPTION
**Pull request by Calanthe**
The Enchilada display card was originally introduces by @uplne few months ago but since then some requirements and designs changed. Back then it was designed to handle both manual (`gu-display`) and cAPI (`gu-comcontent`) formats. Now there will be two separate templates in DFP.

This PR makes changes in the manual card to show properly an ad with a brand's logo, header and text at the bottom of the container, like here in the Nike example:
screen shot 2016-02-16 at 12 06 56

Those values (position of the logo, header, text and font sizes) will be configurable in the DFP template. I will make those changes in the next PR.  

@uplne @jbreckmckye @regiskuckaertz

### CODE REVIEW

File: static/src/javascripts/projects/common/views/commercial/creatives/gu-style-comcontent.html [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
-        <%=data.standFirst%>
-        <div class="gu-brand">
-        	<%=data.noteOrLink%>
-        	<img class="gu-logo" src="<%=data.brandLogo%>" alt="<%=data.brand%>">
-        </div>
+    <div class="gu-mutable gu-content">
+        <a href="<%=data.clickMacro%><%=data.articleUrl%>" class="gu-link" data-link-name="header text">
+            <h2 class="gu-content__title"><%=data.articleHeaderText%></h2>
+        </a>
+        <a href="<%=data.clickMacro%><%=data.articleUrl%>" class="gu-link" data-link-name="text">
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/views/commercial/creatives/gu-style-comcontent.html]: Is it really necessary to have two different `A` elements?

File: static/src/javascripts/projects/common/views/commercial/creatives/gu-style-comcontent.html [hunk 2]
```diff
@@ -1,13 +1,19 @@
-<a class="<%=data.gustyleClass%>" href="<%=data.clickMacro%><%=data.articleUrl%>" data-link-name="creative | gu style commercial content">
-    <div class="u-responsive-ratio inlined-image">
+<div class="gu-display" data-link-name="creative | gu style commercial display content">
+    <div class="gu-image-layer u-responsive-ratio inlined-image">
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/views/commercial/creatives/gu-style-comcontent.html]: Aren't `.gu-display__image-layer` and `.gu-display__image` more idiomatic?

File: static/src/javascripts/projects/common/views/commercial/creatives/gu-style-comcontent.html (hunk 1 above)
Comment (Calanthe) [inline static/src/javascripts/projects/common/views/commercial/creatives/gu-style-comcontent.html]: It's not necessary. But with one `A` when you will hover over a title, the whole text will be underlined. It looks ugly.

File: static/src/stylesheets/module/commercial/gustyle/_gu-comlabel.scss [hunk 3]
```diff
@@ -1,29 +1,35 @@
-.gu-comlabel__btn {
-    @include f-textSans;
-    @include font-size(14, 18);
+.gu-comlabel {
+    .gu-comlabel__btn {
```
Comment (uplne) [inline static/src/stylesheets/module/commercial/gustyle/_gu-comlabel.scss]: Does it need to be nested?

**Comment by uplne** [comment]
One comment, otherwise :+1:

File: static/src/stylesheets/module/commercial/gustyle/_gu-comlabel.scss (hunk 3 above)
Comment (Calanthe) [inline static/src/stylesheets/module/commercial/gustyle/_gu-comlabel.scss]: Yes, because of the specificity I wanted to overwrite some other styles.

**Pull request merged**